routing/index_router.cpp

```diff
@@ -318,13 +318,19 @@ IndexRouter::IndexRouter(VehicleType vehicleType, bool loadAltitudes,
   CHECK(m_directionsEngine, ());
 }
 
-bool IndexRouter::FindBestSegmentInSingleMwm(m2::PointD const & point, m2::PointD const & direction,
-                                             bool isOutgoing, Segment & bestSegment)
+std::unique_ptr<WorldGraph> IndexRouter::MakeSingleMwmWorldGraphForGenerator()
```

Comment:
Maybe it's worth to remove ```ForGenerator``` suffixes?

Reply:
Maybe. But in that case we show explicitly that we uncover WorldGraph for generator only. Beside that we write suffix `ForTesting`. Do you think it's better to remove `ForGenerator` anyway?

Reply:
ForTesting is used for methods that expose somehow internal structure of the class, or methods that are doing something that is very dangerous in prod, but ok in tests.
 
These methods looks good to me, not sure that it's worth to specify that they must be used in generator only. Up to you.

Reply:
Let's remove the suffix.